pkg/windows/windows.go

```diff
@@ -75,8 +75,25 @@ const (
 	serviceNotFound = "status 1060"
 )
 
-// filesToTransfer is a map of what files should be copied to the Windows VM and where they should be copied to
-var filesToTransfer map[*payload.FileInfo]string
+var (
+	// filesToTransfer is a map of what files should be copied to the Windows VM and where they should be copied to
+	filesToTransfer map[*payload.FileInfo]string
+	// RequiredServices is a list of Windows services installed by WMCO
+	RequiredServices = []string{
+		windowsExporterServiceName,
+		kubeProxyServiceName,
+		hybridOverlayServiceName,
+		kubeletServiceName}
```

Comment:
It seems ordering is important here so please add the comment `// This slice order matters due to service dependencies`